static/app/bootstrap/initializeSdk.tsx

```diff
@@ -2,6 +2,7 @@
 import {browserHistory, createRoutes, match} from 'react-router';
 import {ExtraErrorData} from '@sentry/integrations';
 import * as Sentry from '@sentry/react';
+import * as Spotlight from '@spotlightjs/spotlight';
```

Comment:
Can you make this a dynamic import instead?

Reply:
done